src/net/tcp/socket.rs

```diff
@@ -106,6 +106,26 @@ impl TcpSocket {
         sys::tcp::set_linger(self.sys, dur)
     }
 
+    /// Sets the value of `SO_RCVBUF` on this socket.
+    pub fn set_recv_buffer_size(&self, size: u32) -> io::Result<()> {
+        sys::tcp::set_recv_buffer_size(self.sys, size)
+    }
+
+    /// Get the value of `SO_RCVBUF` set on this socket.
+    pub fn get_recv_buffer_size(&self) -> io::Result<u32> {
```

Comment:
In general, I think the convention in Rust is to call getters like this `recv_buffer_size` rather than `get_recv_buffer_size`, but the existing methods had names beginning with `get_`, so I thought it was better to be consistent, especially since it looks like `get_localaddr` and `get_reuseport` [are now stable APIs and can't easily be renamed](https://github.com/tokio-rs/mio/blob/master/CHANGELOG.md#added).

Reply:
You're right the `get_` shouldn't have been a thing, I missed that in reviewing. You can leave it as so it's at least consistent with the `TcpSocket` type, I intend on removing it for v0.8 anyway (#1381).